dataset-spec/dataset-spec.md

```diff
@@ -52,28 +51,14 @@ The coordinate reference system of the values is WGS84 longitude/latitude.
 
 ### Provider Object
 
-The object provides information about a provider. A provider is any of the organizations that created or processed the content of the dataset and therefore influenced the data offered by this dataset.
+The object provides information about a provider. A provider is any of the organizations that captured or processed the content of the dataset and therefore influenced the data offered by this dataset. May also include information about the final storage provider hosting the data.
 
 | Field Name | Type   | Description                                                  |
 | ---------- | ------ | ------------------------------------------------------------ |
 | name       | string | **REQUIRED.** The name of the organization or the individual. |
+| type       | string | The type of provider. Any of `producer`, `processor` or `host`. |
 | url        | string | Homepage of the provider.                                    |
```

Comment:
Perhaps we make a recommendation here that the homepage should ideally include a way to find a contact about the dataset? To help explain a bit why we dropped 'contact' info.

Reply:
Okay, will add that next week.